virtcontainers/network.go

```diff
@@ -1256,7 +1314,7 @@ func deleteNetNS(netNSPath string) error {
 	return nil
 }
 
-func createVirtualNetworkEndpoint(idx int, ifName string, interworkingModel NetInterworkingModel) (*VirtualEndpoint, error) {
+func createVirtualNetworkEndpoint(idx int, ifName, tapName, hwAddr string, interworkingModel NetInterworkingModel) (*VirtualEndpoint, error) {
```

Comment:
We don't need the `tapName` and `hwAddr` here since they have to be generated for veth. Please don't modify this.